modules/inbucket/inbucket.go

```diff
@@ -57,8 +57,12 @@ func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomize
 func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustomizer) (*InbucketContainer, error) {
 	req := testcontainers.ContainerRequest{
 		Image:        img,
-		ExposedPorts: []string{"2500/tcp", "9000/tcp"},
-		WaitingFor:   wait.ForLog("SMTP listening on tcp4"),
+		ExposedPorts: []string{"2500/tcp", "9000/tcp", "1100/tcp"},
+		WaitingFor: wait.ForAll(
+			wait.NewHostPortStrategy("2500/tcp"),
+			wait.NewHostPortStrategy("9000/tcp"),
+			wait.NewHostPortStrategy("1100/tcp"),
```

Comment:
I think this is more readable
```suggestion
			wait.ForListeningPort("2500/tcp"),
			wait.ForListeningPort("9000/tcp"),
			wait.ForListeningPort("1100/tcp"),
```